.ci/clang.sh

```diff
@@ -15,13 +20,14 @@ if [ "$TRAVIS_OS_NAME" = "linux" ]; then
 	export CC=/usr/local/clang-$clang_version/bin/clang
 	symbolizer=/usr/local/clang-$clang_version/bin/llvm-symbolizer
 	export ASAN_SYMBOLIZER_PATH=$symbolizer
+	export MSAN_SYMBOLIZER_PATH=$symbolizer
```

Comment:
Won't the symbolizer path always be the same? So we could just rename `ASAN_SYMBOLIZER_PATH` to `SAN_SYMBOLIZER_PATH` or something like that.